tests/test-verify-volume-mount.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Set some facts
+  set_fact:
+    storage_test_mount_device_matches: "{{ ansible_mounts|selectattr('device', 'equalto', current_volume._device)|list }}"
+    storage_test_mount_point_matches: "{{ ansible_mounts|selectattr('mount', 'equalto', current_volume.mount_point|list }}"
+    storage_test_mount_expected_match_count: "{{ 1 if {{ current_volume.state == 'present' and current_volume.mount_point else 0|int }}"
```

Comment:
Shouldn't you reload facts using the setup module before doing this? Otherwise `ansible_mounts` may contain stale data (collected at the beginning of the playbook). See https://github.com/linux-system-roles/selinux/blob/6cd1ec8fdebdb92a789b14e5a44fe77f0a3d8ecd/tests/selinux_test_transitions.yml#L11. Or is that the intent?

Reply:
The last thing we do in `tasks/main-blivet.yml` is run `setup` to pick up changes we made.

Reply:
That's an interesting approach, maybe we should do that in the other roles too (network and selinux in particular)